### ISSUE TITLE
Async / await

### ISSUE THREAD

**Issue by dgpgdev** [vuejs/vuepress#124]
we can't use async await by default with a custom components in doc 

.vuepress
--components
----customcomponents.vue
```javascript
async sendRequest(){
 const response = await axios.post(this.url, JSON.parse(this.$refs.requestBody.innerText))
  //do soomething
}
```
need a specific loader (babel i suppose).

Maybe a good thing to embed babel async await loader by default

**Comment by ulivz** [comment]
VuePress used `Buble` whose support for **Async/Await** is still in code review: https://github.com/Rich-Harris/buble/pull/125.

**Comment by dgpgdev** [comment]
ok, i changed buble to babel into my fork. Hope coming soon :)

**Comment by dgpgdev** [comment]
done! hope buble implements async await soon :)

**Comment by ycmjason** [comment]
@yyx990803 Just curious, is there any particular reason that we use buble? It seems to lack a lot of essential features, e.g. proper polyfills, async/await... etc.

**Comment by yyx990803** [comment]
@ycmjason Because it's a lighter and faster default and produces a smaller bundle.

**Comment by ycmjason** [comment]
@yyx990803 In an exchange of less-complete functionality :disappointed:

**Comment by ulivz** [comment]
Supported by #336 and merged as 82cd8bdbae6400032d79332302f06c7d196b58f2
